runtime/doc/options.txt

```diff
@@ -2090,6 +2090,17 @@ A jump table for the options with a short description can be found at |Q_op|.
 	based expansion (e.g., dictionary |i_CTRL-X_CTRL-K|, included patterns
 	|i_CTRL-X_CTRL-I|, tags |i_CTRL-X_CTRL-]| and normal expansions).
 
+						*'completefuzzycollect'* *'cfc'*
+'completefuzzycollect' 'cfc'	string	(default: empty)
+				global
+	option enables fuzzy collection in specific |ins-completion| modes,
+	adjusting how items are gathered for fuzzy matching based on input.
+	This option can contain the following values (separated by commas),
+	each enabling fuzzy collection for a specific completion mode:
+	k	keyword completion in 'complete' and current file
+	w	whole lines
```

Comment:
Why not use `l` to be consistent with `<C-x><C-l>` ?

Reply:
good point w is ambiguous l is better